RocketChat/Rocket.Chat
# regression: Check room scoped permissions for slash commands

## Proposed changes (including videos or screenshots)





## Issue(s)


## Steps to test or reproduce


- Create a normal user with role "user"
- Create a channel with this normal user, user should be "owner" of channel.
- User should be able to use "/invite" slash command to add users.

## Further comments

TC-836

Some slash commands were being blocked even after having enough permissions in room scope.
![image](https://github.com/RocketChat/Rocket.Chat/assets/58601732/5281f1a9-b84c-4dea-90d1-61fb6c77ef74)
